examples/examples_test.go

```diff
@@ -37,6 +37,7 @@ func TestExamples(t *testing.T) {
 
 	examples := []integration.ProgramTestOptions{
 		baseJS.With(integration.ProgramTestOptions{Dir: path.Join(cwd, "minimal")}),
+		baseJS.With(integration.ProgramTestOptions{Dir: paht.Join(cwd, "multiple-regions")}),
```

Comment:
I've added this to the "short" CI list for this PR. Happy to move it out before merging.